Firmware/ultralcd.cpp

```diff
@@ -79,7 +79,7 @@ static uint8_t lcd_commands_step = 0;
 CustomMsg custom_message_type = CustomMsg::Status;
 unsigned int custom_message_state = 0;
 
-
+bool isPrintFinished = false;
```

Comment:
Isn't there already some variable with similar meaning/semantics? Adding more of these vars only makes the code more "spaghetti-like"

Reply:
That's the first thing that I have tried to find, but I didn't find any variable that could I use to know that the printer has finished a print job successfully, and because of that I change the status when the card finds the eof. Of course, if there is another existing variable that I could use for that purpose II will suggest the deletion of mine because that doesn't make sense, even more, when the resources are limited like in the 8 bits architecture. Please let me know if there is another vars combination that I could use, anyway I will try to find it harder.

Reply:
How far is card.sdprinting from the semantics of your isPrintFinished?

Reply:
I will check it, the thing is that you need to know if there was a previous print wich was not cancelled in order to offer the reprint to the user.

Reply:
good point, probably not - card.sdprinting is only true when there is some SD print going on
... probably if you rename `isPrintFinished` to `enableReprint` it would be better ... because there is a bunch of isPrintFinished-like vars in the FW and each means something different.

Reply:
Ok, that makes sense. Definitely, I will do the change.

Reply:
The variable **_isPrintFinished_** has been renamed to **_enableReprint_** and the change has been tested successfully.